<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пр4</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 207, 178);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
   <property name="text">
    <string>Проверка криптостойкости паролей</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
   <property name="text">
    <string>  Введите хэш:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_hash">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_from_the_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести из файла</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_hash2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>16</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>16</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
   <property name="text">
    <string>  Выбирите режим:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Однопоточный</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_many">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Многопоточный</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sB_streams">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
   <property name="text">
    <string>Кол-во потоков:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
   <property name="text">
    <string>  Результат:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="tE_result">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(69, 98, 78);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_run">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>510</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(72, 89, 53);</string>
   </property>
   <property name="text">
    <string>Начать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_update">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(72, 89, 53);</string>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_tb">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(72, 89, 53);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
